libminifi/src/core/flow/StructuredConfiguration.cpp

```diff
@@ -900,4 +918,131 @@ void StructuredConfiguration::addNewId(const std::string& uuid) {
   }
 }
 
+void StructuredConfiguration::encryptSensitivePropertiesInYaml(YAML::Node property_yamls, const std::map<std::string, Property>& properties) const {
+  for (auto kv : property_yamls) {
```

Comment:
Minor: this could be const ref

```suggestion
  for (const auto& kv : property_yamls) {
```

Reply:
I see, thanks for the explanation